test/test_images/performance/README.md

```diff
@@ -69,13 +74,13 @@ spec:
           mountPath: /etc/config-mako
       terminationMessagePolicy: FallbackToLogsOnError
     - name: aggregator
-      image: knative.dev/eventing/test/test_images/latencymako
+      image: knative.dev/eventing/test/test_images/performanceaggregator
```

Comment:
Does this command exist? Looks like there's just test/test_images/performance.

Reply:
Looks like intellij refactored this string too :smile: